net/scripts/install-at.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#
+set -ex
+
+[[ $(uname) = Linux ]] || exit 1
+[[ $USER = root ]] || exit 1
+
+apt-get update
+apt-get --assume-yes install at
```

Comment:
are we installing `at` since it is no longer in the base image of ubuntu 22?

Reply:
yep, exactly.